### PR TITLE
Fixed support beams to only be placeable on beams at least three blocks high

### PR DESCRIPTION
**Pull request by DavidTriphon**
What it says on the tin.

### CODE REVIEW

**Comment by Fr4gtastic** [comment]
To be clear — is it only three błocks or at least three błocks?

**Comment by DavidTriphon** [comment]
At least. It only checks the two blocks beneath the attachment block. Beyond that it doesn’t care if it is solid or support.

**Comment by DavidTriphon** [comment]
The picture shows beams placed on 3, 4, and 5 height beams but not beneath that.

**Comment by Fr4gtastic** [comment]
I see, thanks for the clarification.

File: src/main/java/net/dries007/tfc/objects/blocks/wood/BlockSupport.java [hunk 1]
```diff
@@ -292,10 +292,8 @@ public boolean canPlaceBlockOnSide(@Nonnull World world, @Nonnull BlockPos pos,
         }
         else
         {
-            if (!isConnectable(world, pos, side.getOpposite()))
-            {
-                return false;
-            }
+            if (!isConnectable(world, pos, side.getOpposite())) return false;
+            if (!isThreeTall(world, pos.offset(side.getOpposite()))) return false;
             int distance = getHorizontalDistance(side, world, pos);
```
Comment (DisasterMoo) [inline src/main/java/net/dries007/tfc/objects/blocks/wood/BlockSupport.java:296]: This is only checking on one side. Better to implement that logic on `getHorizontalDistance` and only return a value if both sides are at least three tall.

**Review by DisasterMoo — CHANGES_REQUESTED** [review]

**Comment by DavidTriphon** [comment]
Fixed 3 tall check to work on both sides and moved 3 tall check to the `getHorizontalDistance` method.

**Pull request merged**